CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# Contributing to dask-awkward
+
+Install the development environment (once) with
+
+```bash
+uv venv
+uv pip install -e '.[io,docs,test]'
+pre-commit install
+```
+
+Source development environment with
+
+```bash
+source .venv/bin/activate
```

Comment:
Doesn't this need to come before pip install?

Reply:
In general yes, but not with `uv`. If it finds a venv (even though it is not activated) it will install software into it.

Reply:
OK, so if we remove requiring `uv`, then this will need to be reordered. That's conceptually cleaner anyway, if you ask me.